config/locales/email_subscribers.es.yml

```diff
@@ -0,0 +1,8 @@
+es:
+  email_subscribers:
+    form:
+      subscribe_description: "Suscríbete para recibir un resumen semanal de las publicaciones en tu bandeja de entrada"
+      email_placeholder: "Ingresa tu email..."
```

Comment:
In Spain, I’d say “Introduce tu email” rather than “Ingresa tu email”, but the latter might be more common in other countries.